src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
```

Comment:
> Content is vertically centered (for any header height)
![image](https://github.com/mate-academy/layout_moyo-header/assets/45945732/2d290d76-d3ec-4bd9-b3f0-b1df35641006)
I believe logo also should be centered